### ISSUE TITLE
Double/no indentation when closure and parentheses on same line

### ISSUE THREAD

**Issue by ChristopherRogers** [nicklockwood/SwiftFormat#538]
The following (formatted) code gets formatted oddly with `--disable trailingClosures`. The indentation inside of the closure is double indented (one for the parentheses, I assume). The indentation for the last line gets completely ignored.

Tested in 0.44.0

```swift
let a = b.flatMap({ e
        in
    e
                                                 })
```

```swift
let a
    = b.flatMap({ e
            in
        e
                                                })
```

**Comment by nicklockwood** [comment]
@ChristopherRogers this is slightly improved in 0.44.15. Result now looks like:

```swift
let a = b.flatMap({ e
        in
    e
})
```

**Comment by nicklockwood** [comment]
@ChristopherRogers fixed in 0.45.5.
